static/src/javascripts/projects/common/modules/atoms/quiz.js

```diff
@@ -0,0 +1,79 @@
+define([
+    'common/utils/$',
+    'lodash/collections/forEach'
+], function (
+    $,
+    forEach
+) {
+    var $quizzes = $('.quiz--questions');
+    var quizAnswerLabelClassName = 'quiz__answer';
+    var quizAnswerLabelCheckedClassName = quizAnswerLabelClassName + '--checked';
+    var quizAnswerLabelFocussedClassName = quizAnswerLabelClassName + '--focussed';
+    var quizAnswerInputClassName = quizAnswerLabelClassName + '__input';
+
+    function renderLabels(state) {
+        forEach(state.labels, function (label) {
```

Comment:
I'm happy with (your) original code, it's idiomatic and doesn't require the reader to understand the context of execution, ie. is enhanced or not.